src/components/StyledText/index.js

```diff
@@ -62,7 +65,7 @@ export function StyledTextPure({
   }
 
   return (
-    <Wrapper className={`styledText ${fontClass}`} style={style}>
+    <Wrapper className={`styledText ${fontClass} ${predefinedStyle}`} style={style}>
```

Comment:
`predefinedStyle` is being output as `undefined`. Maybe set it to an empty string if it's not found? That way you won't have to work around conditionally rendering it out as a class that is an empty string will just add an empty string to the classes.

I'm not sure how you want to go about handling this. Changing the function to return an empty string instead of `undefined` might break elsewhere and we don't have tests written for it yet.

Maybe just do an OR check afterwards? I haven't tested it, but it should work as intended.

```js
 let predefinedStyle = getFirstValue(fields?.predefinedStyle) || '';
 ```